pkg/storage/cedar_system_bookmark_test.go

```diff
@@ -3,16 +3,14 @@ package storage
 import (
 	"context"
 
-	"github.com/google/uuid"
-
 	"github.com/cmsgov/easi-app/pkg/models"
 )
 
 func (s StoreTestSuite) TestCreateCedarSystemBookmark() {
```

Comment:
Now that we have the functionality, can you make a test to cover inserting duplicate bookmarks? Just want to have a test to cover it so we don't accidentally change anything on it later 😁

Reply:
Will do

Reply:
@ClayBenson94 I have added that test to my latest commit. It creates two of the same bookmarks, and verifies that the created_at is updated